Libraries/Storage/AsyncStorage.js

```diff
@@ -20,6 +19,14 @@ const RCTAsyncStorage =
   NativeModules.AsyncSQLiteDBStorage ||
   NativeModules.AsyncLocalStorage;
 
+type MultiRequestType = {|
+  keys: Array<string>,
+  callback: ?(errors: ?Array<Error>, result: ?Array<Array<string>>) => void,
+  keyIndex: number,
+  resolve: ?(result?: Promise<any>) => void,
```

Comment:
Why does this Promise resolve the `any` type? Are we able to be more specific with this type?